documents/operating.md

```diff
@@ -11,12 +12,83 @@ Contents:
 6.  Policies
 
 ---------------------------------------------------------------------
+SECTION 0: 📓📓📓 Definitions 📓📓📓
+---------------------------------------------------------------------
+
+* **Bad Idea Factory Limited Liability Company ("the Company")** is a Limited Liability Company domiciled in Pennsylvania.
+* The Company may also be referred to as "**Bad Idea Factory**" or "**BIFFUD**".
+* "**Corporate Overlord**" shall mean an "owner" of the Company under Pennsylvania state law.
+* "**Associate**" shall mean any person legally associated with the Company through contract or ownership.
+* The "**Hive Mind**" shall mean the "board of directors" of the Company, as defined by Pennsylvania state law. The Hive Mind shall consist of all current Corporate Overlords of the Company.
+* "**Plotting Session**" shall mean a meeting of the "board of directors" of the Company, as defined by Pennsylvania state law.
+* "**Quorum**" shall mean the presence of 2/3 or more of the company's Corporate Overlords.
+* "Operating Agreement" shall refer to the official version of this document
+to be held in a file named `operating.md` in the `documents` directory of the
+`corporate` repository within the `BadIdeaFactory` team hosted on GitHub.
+
+### Support Categories
+* "**Complete Consensus**" shall mean 100% support or more.
+* "**Strong Consensus**" shall mean 2/3 support or more.
+* "**Majority**" shall mean over 1/2 support.
+* "**Apathetic Support**" shall mean over 1/3 vocalized support and less than 1/3 vocalized dissent.
+
+---------------------------------------------------------------------
 SECTION 1: 📚📚📚 Official Operating Agreement 📚📚📚
 ---------------------------------------------------------------------
-- [ ] Company term
-- [ ] Continuance of company
-- [ ] Rickroll
-- [ ] Game of Mario Party 3 before legal arbitration between members
+- [x] Company term
+- [x] Continuance of company
+- [x] Rickroll
+- [x] Game of Mario Party 3 before legal arbitration between members
+
+### Company Term
+BIFFUD will exist until either (1) a vote to terminate the organization
+is passed by a consensus of all Corporate Overlords or (2) the Hive Mind fails to
+hold a single meeting with quorum for a period of a consecutive year,
+(3) everyone dies / there are no more Corporate Overlords, or (4) if all
+Corporate Overlords are straight white cis men.
```

Comment:
(4 is one of my favorite things about this entire document) -- TBH I'm curious what the lawyer will say, but I think this is worth including for sure since it speaks directly to the values of the organization.

I'm comfortable trying to define things to be more legally explicit -- we mean ethnicity and sexual orientation.  Someone could claim to be LGBTQ and that would fulfill the terms of (4).

I don't see this clause ever taking effect -- it's function is 99% the fact that it exists.